### ISSUE TITLE
Ability to specify custom URL for card

### ISSUE THREAD

**Issue by darkhelmet** [ToyDragon/TTSDeckImporter#3]
This would be super useful when spoilers are released and cards aren't available on magiccards.info yet. I mean sure you can just jam in a different card, but the joy of this stuff is that you **can** have the _real_ card.

If you like this idea and imagine it wouldn't be terribly difficult, point me in the right direction of where to start and I can make a PR.

**Comment by ToyDragon** [comment]
I don't think it would be too difficult to implement, but I really don't want to let people upload arbitrary images to my servers. Currently I allow people to upload arbitrary images as the hidden card image, but I'm considering removing that too. 

If you want to implement it yourself though go for it, as long as their is a way to turn it off in the config somehow I will accept the PR. You could add a field to the [Card class](https://github.com/ToyDragon/TTSDeckImporter/blob/07b96707bf1b720ec49c98ab7390ffdbc9e3e1d1/libs/src/core/Card.java) that contains a source URL, populate it in the [ReadCard](https://github.com/ToyDragon/TTSDeckImporter/blob/07b96707bf1b720ec49c98ab7390ffdbc9e3e1d1/libs/src/core/DeckMaker.java#L195) method of DeckMaker.java, create a [Card Retriever](https://github.com/ToyDragon/TTSDeckImporter/tree/master/libs/src/utils/cardretrieval) class that downloads the source image and populates card.imageFileName, and add that retiever to the list in [ImageUtils.java](https://github.com/ToyDragon/TTSDeckImporter/blob/56c753229e674402074fe9cb166576a8895f692d/libs/src/utils/ImageUtils.java#L35). I don't think you would need any special handling anywhere else, you may want to make sure that the other card retrievers don't step on your toes and use standard images if you populate the name or something.

If you have any issues setting up the project feel free to shoot me an email or something.
